lab_1_classify_by_unigrams/start.py

```diff
@@ -14,7 +15,21 @@ def main() -> None:
         de_text = file_to_read_de.read()
     with open("assets/texts/unknown.txt", "r", encoding="utf-8") as file_to_read_unk:
         unknown_text = file_to_read_unk.read()
-    result = None
+    print(tokenize(en_text))
+    print(create_language_profile("en", en_text))
+    en_profile = create_language_profile("en", en_text)
+    unk_profile = create_language_profile("unk", unknown_text)
+    de_profile = create_language_profile("de", de_text)
+    if (isinstance(en_profile, dict) and isinstance(de_profile, dict)
```

Comment:
do you trust to your code? why isinstance in start.py?

Reply:
because mypy thinks i shouldn't trust my code :( i'll try to fix it some other way then

Reply:
fixed